src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -940,6 +977,14 @@ def SetBlockingCB(self, blockingCB):
 
         self._ChipStack.blockingCB = blockingCB
 
+    def IssueNOCChain(self, csr: Clusters.OperationalCredentials.Commands.CSRResponse, nodeId: int):
```

Comment:
How exactly are clients supposed to use this? We should be either making this an async method that would await and return the resultant NOC chain, OR, you permit callers to pass in a callback of their own.

Reply:
Also, comments please on the method.

Reply:
I stand corrected. `CallAsync` does indeed _wait_ until the `pychip_DeviceController_IssueNOCChain` has finished executing on the chip thread, AND, it waits until the cond var `completeEvent` is indeed set, and returns back `callbackRes`.

The naming here is unfortunately somewhat poor (and the documentation equally so). But the logic here is fine.